ethereum/evmtool/src/test/resources/org/hyperledger/besu/evmtool/state-test/blockhash.json

```diff
@@ -61,7 +61,7 @@
       "post": {
         "Shanghai": [
           {
-            "hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "hash": "0xd14c10ed22a1cfb642e374be985ac581c39f3969bd59249e0405aca3beb47a47",
```

Comment:
is it normal to change that ?

Reply:
yes, bonsai cares about state root, whereas forest does not.  This value was just a placeholder afaik.